README.md

```diff
@@ -40,43 +40,66 @@ With Griptape, you can create *structures*, such as `Agents`, `Pipelines`, and `
-         Action: {"type": "tool", "name": "WebScraper", "activity": "get_content", "input": {"values": {"url": "https://www.griptape.ai"}}}
-INFO     Subtask 13a7772aea2840259c521f6dcaf8a7b0
-         Observation: Output of "WebScraper.get_content" was stored in memory with memory_name "TextToolMemory" and artifact_namespace "a84ab9258aa540f99607c5df73da2d2d"
-INFO     Subtask 905bc561be3f4ff48894d8984814a44f
-         Thought: Now that the webpage content is stored in memory, I can use the TextToolMemory's summarize activity to summarize the content.
-         Action: {"type": "memory", "name": "TextToolMemory", "activity": "summarize", "input": {"values": {"memory_name": "TextToolMemory", "artifact_namespace": "a84ab9258aa540f99607c5df73da2d2d"}}}
-INFO     Subtask 905bc561be3f4ff48894d8984814a44f
-         Observation: The text describes Griptape, an open-source Python framework and managed cloud platform for building and deploying AI applications for enterprise use. Griptape allows users to easily create AI-powered agents, compose event-driven pipelines,
-         and orchestrate complex workflows. It provides tools for connecting to data sources securely and controlling data access. The text also provides examples of creating a Griptape agent, pipeline, and workflow. Griptape Cloud is mentioned as a managed
-         platform for running AI agents, pipelines, and workflows. The text concludes by offering customers the opportunity to contact Griptape for further information and assistance.
-INFO     Subtask 459f35e5af4e49f7ac87a5989447dcb4
-         Thought: Now that I have summarized the content of the webpage, I can store this summary in a file named "griptape.txt" using the FileManager tool's save_content_to_file activity.
-         Action: {"type": "tool", "name": "FileManager", "activity": "save_content_to_file", "input": {"values": {"path": "griptape.txt", "content": "The text describes Griptape, an open-source Python framework and managed cloud platform for building and deploying
-         AI applications for enterprise use. Griptape allows users to easily create AI-powered agents, compose event-driven pipelines, and orchestrate complex workflows. It provides tools for connecting to data sources securely and controlling data access. The
-         text also provides examples of creating a Griptape agent, pipeline, and workflow. Griptape Cloud is mentioned as a managed platform for running AI agents, pipelines, and workflows. The text concludes by offering customers the opportunity to contact
-         Griptape for further information and assistance."}}}
-INFO     Subtask 459f35e5af4e49f7ac87a5989447dcb4
-         Observation: saved successfully
-INFO     ToolkitTask 51b46eff74a64133a1b6d47c630f1db5
-         Output: The summarized content of the webpage "https://www.griptape.ai" has been successfully stored in a file named "griptape.txt".
+[11/02/23 15:28:24] INFO     ToolkitTask 72b89a905be84245a0563b206795ac73       
```

Comment:
I had manually removed the timestamp to make the example a little more "timeless".